x/axelarcork/types/msgs.go

```diff
@@ -0,0 +1,149 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var (
+	_ sdk.Msg = &MsgScheduleAxelarCorkRequest{}
+)
+
+const (
+	TypeMsgScheduleCorkRequest = "cork_schedule"
```

Comment:
We may also want to add "axelar" to these message strings.

Reply:
done